src/main/java/dev/jbang/cli/Edit.java

```diff
@@ -42,11 +43,12 @@
 	boolean live;
 
 	@CommandLine.Option(names = {
-			"--open" }, description = "Opens editor/IDE on the temporary project.")
-	String editor;
+			"--open" }, description = "Opens editor/IDE on the temporary project.", fallbackValue = "${JBANG_EDITOR:-}", preprocessor = StrictParameterPreprocessor.class)
+	Optional<String> editor;
```

Comment:
Just curious: is the Optional necessary here or did you just like it better this way?

Reply:
New feature of picocli that makes it possible to use optional to signal if option was present or not on the command line - no matter if value is null or default. 

Technically could still use null but I liked it better to be explicit andntypesafe

Reply:
Sure. Then we should just gradually convert the rest of the optional options as well. For consistency sake :-)